analytic_engine/src/table/data.rs

```diff
@@ -128,10 +133,11 @@ pub struct TableData {
     /// Allocating memtable id should be guarded by write lock
     last_memtable_id: AtomicU64,
 
-    /// Last id of the sst file
-    ///
-    /// Write to last_file_id require external synchronization
-    last_file_id: AtomicU64,
+    /// Max id of the sst file
```

Comment:
```suggestion
    /// Max id of the sst file.
   ///
   /// The id is allocated by step, so there are some still unused ids smaller than the max one. And this field is only a mem state for Manifest, it can only be updated during recover or by Manifest.
```